ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -625,6 +625,40 @@ class SignalSpec: QuickSpec {
 				observer.sendNext("d")
 				expect(values) == [ "a", "cc", "d" ]
 			}
+
+			it("should not store strong reference to previously passed items") {
+				var disposedItems: [Bool] = []
+
+				struct Item {
```

Comment:
No you can keep the implementation as is, I just think it would be clearer if the container type (`Item`) had reference semantics itself.